crates/extensions/c8y_mapper_ext/src/operations/handlers/config_update.rs

```diff
@@ -120,7 +120,7 @@ mod tests {
                 "te/device/main///cmd/config_update/c8y-mapper-123456",
                 json!({
                     "status": "init",
-                    "remoteUrl": "http://www.my.url",
+                    "remoteUrl": "http://www.my.url/",
```

Comment:
It would be better to understand why the new method `local_proxy_url` is adding a trailing `/`.

Reply:
In accordance with the [URL spec](https://github.com/servo/rust-url/issues/808), `Url::parse` by default adds a `/` character after the host if the path is empty.
In our case we parse the URL and use a `.to_string()` here:

https://github.com/thin-edge/thin-edge.io/pull/3224/files#diff-6806117b091373774a361e1c5eaf2ba17e029724012bc70656e85fe99b2c87cdR271